docs/maker-cli.md

```diff
@@ -166,90 +174,104 @@ This shows the server tor address, which is our maker server's identity on the T
 ---
 
 ### ShowFidelity
+
 When setting up `makerd`, we fund the maker's wallet and create a fidelity bond. To see details about our existing fidelity bond, use:
 
 ```bash
 $ ./maker-cli show-fidelity
 ```
 
-**Output:**  
-```bash
-{
-    0: (
-        FidelityBond {
-            outpoint: OutPoint {
-                txid: 6c06a925066b0cf8adb400e53001b20587729407bce7dcb95dcacd038950b0e4,
-                vout: 0,
-            },
-            amount: 50000 SAT,
-            lock_time: 2465 blocks,
-            pubkey: PublicKey { ... },
-            conf_height: 229349,
-            cert_expiry: 5,
-        },
-        false,
-    ),
-}
+**Output:**
+
+```json
+[
+  {
+    "amount": 50000,
+    "bond_value": 0,
+    "index": 0,
+    "outpoint": "6c06a925066b0cf8adb400e53001b20587729407bce7dcb95dcacd038950b0e4:0",
+    "status": "Live"
+  }
+]
 ```
 
-This shows our maker's fidelity bond. 
+This shows our maker's fidelity bond in a clean JSON format:
 
-> **Note:** Currently, a maker can have only one active (unexpired) fidelity bond at a time. Once a bond expires and is redeemed, a new fidelity bond can be created.
+- **amount**: The amount locked in the fidelity bond (50,000 sats in this example)
+- **bond_value**: The calculated bond value (0 indicates a newly created bond)
+- **index**: The bond index (0 for the first/current bond)
+- **outpoint**: The transaction output point (txid:vout) where the bond is locked
+- **status**: Current status of the bond ("Live" means active and unexpired)
 
+> **Note:** Currently, a maker can have only one active (unexpired) fidelity bond at a time. Once a bond expires and is redeemed, a new fidelity bond can be created.
 
 ---
 
 ### ListFidelityUTXOs
+
 To view fidelity UTXOs in the maker's wallet, run:
 
 ```bash
 $ ./maker-cli list-utxo-fidelity
 ```
 
-**Output:**  
+**Output:**
+
 ```bash
 [
     ListUnspentResultEntry {
-        txid: 6c06a925066b0cf8adb400e53001b20587729407bce7dcb95dcacd038950b0e4,
+        txid: be886c157672f45b4944ee46e973f73adfb1e3a48c1aa62f368daf77e861d064,
         vout: 0,
         address: Some(
-            Address<NetworkUnchecked>(BCRT1QKP92002WJPU5WFLU0YNTU3YXRVDR52N98STWFZJ7MF3F88RJ5PLQKDEUQY),
+            Address<NetworkUnchecked>(TB1QTTUTR6NUUM6E5NEYDDUKXRZVNX87EKSTEU9VZX6XFMRFC30CPPQSPA6UT2),
+        ),
+        label: Some(
+            "f4589480",
         ),
+        redeem_script: None,
+        witness_script: None,
+        script_pub_key: Script(OP_0 OP_PUSHBYTES_32 5af8b1ea7ce6f59a4f246b79630c4c998fecda0bcf0ac11b464ec69c45f80841),
```

Comment:
We need to fix this. Just like taker app, the this utxo data to be translated into `UTXO` struct. 

This can be done the `maker::rpc::server` where the utxo list is processed as sent back to the maker-cli. Then in the maker-cli we will use `serde_json::to_string_pretty` to pretty print the utxo struct in json. 

Then this documentation will be updated with new utxo struct.